internal/constants/get_daytona_script_test.go

```diff
@@ -0,0 +1,22 @@
+//go:build testing
+
+// Copyright 2024 Daytona Platforms Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package constants
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetDaytonaScript(t *testing.T) {
+	BaseUrl := "https://download.daytona.io/daytona"
```

Comment:
Because `https://download.daytona.io/daytona` is already contained in the script, this test will always pass.

Please use a different URL for the test.

Also, please start variable names with a lowercase letter